website/docs/r/env.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+subcategory: "core/v1"
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_env"
+description: |-
+  The resource provides mechanisms to inject containers with environment variables 
+  that are managed specifically by Terraform.
+---
+
+# kubernetes_env
+
```

Comment:
I think you accidentally deleted the line that was supposed to be here. We need a short description here.